vnext/Microsoft.ReactNative/Modules/NativeUIManager.cpp

```diff
@@ -928,9 +932,10 @@ void NativeUIManager::SetLayoutPropsRecursive(int64_t tag) {
     float top = YGNodeLayoutGetTop(yogaNode);
     float width = YGNodeLayoutGetWidth(yogaNode);
     float height = YGNodeLayoutGetHeight(yogaNode);
-    auto view = shadowNode.GetView();
-    auto pViewManager = shadowNode.GetViewManager();
-    pViewManager->SetLayoutProps(shadowNode, view, left, top, width, height);
+    const auto uiElement = view.try_as<xaml::UIElement>();
+    if (!isCollapsed) {
+      pViewManager->SetLayoutProps(shadowNode, view, left, top, width, height);
+    }
```

Comment:
Seems like this should have a comment as to why we don't update the layout props. As in this is not a matter of correctness but is a performance optimization to reduce layout invalidations.